modeling-cmds/src/def_enum.rs

```diff
@@ -1073,7 +1073,21 @@ define_modeling_cmd_enum! {
         #[derive(
             Clone, Debug, Deserialize, JsonSchema, Serialize, ModelingCmdVariant,
         )]
-        pub struct DefaultCameraCenterToSelection;
+        pub struct DefaultCameraCenterToSelection {
+            /// If true actually moves the camera position based on its distance from the current center.
+            /// Otherwise, the camera will orbit around the new center from its current position
+            pub adjust_vantage: bool,
```

Comment:
This should be an enum, with two variants `AdjustVantage` and `OrbitAroundNewCenter`. I expect we'll add a third variant as time goes on, which would be breaking change if you use a bool here, but is easy if we use an enum.

Reply:
oh i see what you're saying.  boolean -> enum with two cases.  sounds good.  I'd like it to be defaulable too to the current behavior.